Temporary patch to load plugins from $CONFIG_FOLDER/plugins/,
usually $HOME/.config/xournalpp/plugins/

Otherwise, the only way to load plugins is to place them in the
system directory /usr/share/xournalpp/plugins.

A more general approach for plugin paths is planned, see
https://github.com/xournalpp/xournalpp/issues/1155#issuecomment-623234420

--- a/src/plugin/PluginController.cpp	2021-03-14 00:26:17.633925344 -0300
+++ b/src/plugin/PluginController.cpp	2021-03-14 00:27:08.487299784 -0300
@@ -15,6 +15,7 @@
 PluginController::PluginController(Control* control): control(control) {
 #ifdef ENABLE_PLUGINS
     auto searchPath = control->getGladeSearchPath()->getFirstSearchPath();
+    loadPluginsFrom(Util::getConfigSubfolder("plugins"));
     loadPluginsFrom((searchPath /= "../plugins").lexically_normal());
 #endif
 }
